tests/parameter_search/test_p_error_binary_search.py

```diff
@@ -33,17 +31,17 @@
 # For built-in models (trees and QNNs) we use the fixture `load_data`
 # For custom models, we define the following variables:
 DATASETS_ARGS = {
-    "FashionMNIST": {
-        "dataset": datasets.FashionMNIST,
-        "train_transform": transforms.Compose(
-            [transforms.ToTensor(), transforms.Normalize(0.2859, 0.3530)]
-        ),
-        "test_transform": transforms.Compose(
-            [transforms.ToTensor(), transforms.Normalize(0.2859, 0.3530)]
-        ),
-        "dataset_size": 60000,
-        "n_classes": 10,
-    },
+    # "FashionMNIST": {
```

Comment:
why not remove it entirely ?

Reply:
Because it's the one causing all the flakies

Reply:
Looks like I'm missing coverage, I'll continue working on it on monday

Reply:
Ho you mean why I commented it, it's because it's a draft, I'll remove in the final PR

Reply:
you still want to remove it right ?